code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -814,8 +814,8 @@
 
 	if(isjellyperson(H))
 		to_chat(H, "<span class='warning'>Your jelly shifts and morphs, turning you into another subspecies!</span>")
-		var/species_type = pick(subtypesof(/datum/species/jelly))
-		H.set_species(species_type)
+		var/species_category = pick(subtypesof(/datum/species/jelly))
+		H.set_species(species_category)
```

Comment:
i dont think species category is the right change here, assuming you did a replace all on it and it caught this, as it's an entirely different thing to species type